### DISCUSSION TITLE
Improvement Suggestions

### DISCUSSION THREAD

**Discussion by daallgeier** [stopthessotax/sso-wall-of-shame#46] [General]
We really should add the Contributors.md from the parent repository and maybe a link to [something like this](https://www.omnicalculator.com/math/percentage-increase) to make it easier for submitters.

_Originally posted by @mbainter in https://github.com/stopthessotax/sso-wall-of-shame/pull/43#discussion_r848807612_

**Comment by daallgeier** [comment]
@mbainter I added this, actually.  So would be great to have some improvements here too.

**Comment by daallgeier** [comment]
There has also been a pull request done a couple of days ago to the original one which splits the vendors into a file each.  This looks more maintainable.  https://github.com/robchahin/sso-wall-of-shame/tree/master/_vendors

**Reply by mbainter** [reply]
I noticed it became active again.  I really wish there weren't two projects, but given that Rob still seems not to want to open it up I don't think it can be avoided.

I'm not sure how much difference it makes to split it into multiple files for something like this, but I'm not opposed.

**Comment by daallgeier** [comment]
I think keeping to the same structure helps when trying to push a pull
request back to rob.

Same with if we pull from there.

Having a file each will also help with creating new pull requests (from
Issues) as one doesn't have to go into the file, look for the right
position, add the content, and then create the pull request.

**Reply by grbray** [reply]
I would love to have them separated so I can do a PR here and not impact the rest of the list.  Helps avoid merge conflicts too.
